app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -54,7 +54,6 @@
         android:scrollbars="none"
         android:paddingBottom="168dp"
         android:clipToPadding="false"
-        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Other constraints need to be removed as well.

Reply:
done